runtime/roots_byt.c

```diff
@@ -122,16 +122,12 @@ CAMLexport void caml_do_local_roots (scanning_action f, value *stack_low,
   int i, j;
 
   for (sp = stack_low; sp < stack_high; sp++) {
-#ifdef NO_NAKED_POINTERS
```

Comment:
See above. The compactor should work with the page table even if the heap or roots contain naked pointers. So this change is not necessary.

Reply:
I agree with @jhjourdan that the `#ifdef NO_NAKED_POINTERS` must stay, at least until the "naked pointers" mode disappears.

Reply:
restored